src/analysis/utility/results.py

```diff
@@ -0,0 +1,194 @@
+import datetime
+from pathlib import Path
+import importlib.resources
+from reportlab.pdfgen import canvas
+
+from reportlab.lib.pagesizes import letter
+from reportlab.pdfgen import canvas
+from reportlab.lib import colors
+from reportlab.platypus import SimpleDocTemplate, Paragraph, Spacer
+from reportlab.lib.styles import getSampleStyleSheet
+from PyPDF2 import PdfReader, PdfWriter
+from reportlab.platypus import SimpleDocTemplate, Table, TableStyle
+
+## THIS WILL BE REMOVED
+## BEING KEPT HERE JUST IN CASE
+
+def get_output_directory():
+    # Get the current script's directory
+    base_dir = Path(__file__).parent.parent
+
+    output_dir = base_dir / 'output'
+    # Create the directory if it doesn't exist
+    output_dir.mkdir(parents=True, exist_ok=True)  
+
+    return output_dir
+
+def create_pdf(title="Analysis Report"):
```

Comment:
I think we discussed this in meeting, but we can probably just print a summary and output graph png/dotviz if you need